service_crategen/src/main.rs

```diff
@@ -53,7 +53,9 @@ fn main() {
                 .long("outdir")
                 .short("o")
                 .takes_value(true)
-                .required(true)))
+                .required(true))
+            .arg(Arg::with_name("no_format")
```

Comment:
Again why the no format here?

Reply:
This was really helpful for iterating quickly on codegen changes, since it massively reduces the time for codegen + typecheck. If you don't mind, I'd like to leave it at least an option (it still formats by default), and maybe we could also add it as a tip to `CONTRIBUTING.md`.